### ISSUE TITLE
Table Accessibility

### ISSUE THREAD

**Issue by Rek3650** [RITAccess/data-analyzer#8]
While trying to navigate the page using only keyboard controls I noticed I was unable to enter the table. I am not sure if I'm missing some controls or if its lacking tab functionality.

**Comment by baozi90** [comment]
This might have something to do with the CopyPasteDiv div.

**Comment by erl7902** [comment]
Changing to slickTable - currently only partially tabbable (becomes a keyboard trap at the end of the table) but should be fixable

**Comment by erl7902** [comment]
Still a keyboard trap - should be fixed after a merge with the new frontEnd changes

**Comment by smcintosh881** [comment]
In order to get out of the table "Option + Tab" has to be used. I couldn't find this in the VoiceOver commands list. 

Another thing I noticed with the table is that when you go to each cell VoiceOver reads the whole chart. Looking into it to see if there is a way for it to just read the data in one cell, not the whole chart.

**Comment by smcintosh881** [comment]
When table was changed to slicktable this issue went away. Use tab to enter the table and control or option tab depending on computer
